source/administration/minio-console.rst

```diff
@@ -28,6 +28,28 @@ You can use the MinIO Console for administration tasks like Identity and Access
 The MinIO Console is embedded as part of the MinIO Server. 
 You can also deploy a standalone MinIO Console using the instructions in the :minio-git:`github repository <console>`.
 
+Supported Browsers
+~~~~~~~~~~~~~~~~~~
+
+MinIO Console runs on a variety of current, stable release browsers.
+
+MinIO recommends use the latest stable release of your preferred browser for the best experience in the MinIO Console.
+Some browsers that are supported include:
+
+- Chrome
+- Edge
+- Safari
+- Firefox
+- Opera
+
+This list is *not* exhaustive and is subject to change.
+
+For a full list of browsers and versions for running MinIO Console, see `Browserslist <https://browsersl.ist/#q=%3E0.2%25%2Cnot+dead+and+not+op_mini+all>`__.
```

Comment:
Perhaps not relevant comment: I'm not sure what to make of `Browserslist` as a word. If you don't know what that is (hi :wave:) it looks like a typo.

Maybe use its domain name instead? `Browsersl.ist`? I'm not convinced that is much better, but at least there's a clue this is talking about a website.

Reply:
Trying a different approach.